UPGRADE-3.x.md

```diff
@@ -1,28 +1,35 @@
 UPGRADE 3.x
 ===========
 
+UPGRADE FROM 3.x to 3.x
+=======================
+
+### Support for NelmioApiDocBundle > 3.6 is added
+
+Since version 3.x, support for nelmio/api-doc-bundle > 3.6 is added. Controllers for NelmioApiDocBundle v2 were moved under `Sonata\*Bundle\Controller\Api\Legacy` namespece and controllers for NelmioApiDocBundle v3 was added as replacement. If you extend them, you must ensure they are using the corresponding inheritance.
```

Comment:
```suggestion
Since version 3.x, support for nelmio/api-doc-bundle > 3.6 is added. Controllers for NelmioApiDocBundle v2 were moved under `Sonata\*Bundle\Controller\Api\Legacy\` namespace and controllers for NelmioApiDocBundle v3 were added as replacement. If you extend them, you must ensure they are using the corresponding inheritance.
```